src/main/scala/zio/s3/MultipartUploadOptions.scala

```diff
@@ -0,0 +1,35 @@
+package zio.s3
+
+import software.amazon.awssdk.services.s3.model.CreateMultipartUploadRequest
+import zio.s3.MultipartUploadOptions.MinMultipartPartSize
+
+import scala.jdk.CollectionConverters._
+
+/**
+ * The upload options that are specific to multipart uploads
+ *
+ * @param uploadOptions [[UploadOptions]]
+ * @param partSize the size of the part in bytes, the minimum is 5 MB
+ * @param parallelism the number of parallel requests to upload chunks, default to 1
+ */
+case class MultipartUploadOptions(
+  uploadOptions: UploadOptions = UploadOptions.default,
+  partSize: Int = MinMultipartPartSize,
+  parallelism: Int = 1
+)
+
+object MultipartUploadOptions {
+  final val MinMultipartPartSize: Int = 5 * 1024 * 1024 // 5 MB
```

Comment:
could you please rename MinChunkSize

Reply:
https://github.com/zio/zio-s3/pull/117/commits/bda1d842c0e9c8f75efb4e3e1084fc3270b79a96#diff-362a387518f51a267b646add5682a0d6R33